lib/is-flaky.js

```diff
@@ -7,9 +7,10 @@ var distro = lsb['DISTRIB_ID'] || '';
 var release = lsb['DISTRIB_RELEASE'] || '';
 var version = process.version;
 var platform = [process.platform, process.arch].join('-');
+var endian = process.config.variables.node_byteorder;
 
 function isStringFlaky(flaky) {
-  var checks = [distro, release, version, platform];
+  var checks = [distro, release, version, platform, endian];
```

Comment:
can you add a check for `endian` before it is added to the array please. I don't believe that `process.config.variables.node_byteorder` exists on v0.10 and v0.12

Could you also add a TODO that this can be removed when support for v0.10 and v0.12 is dropped